.github/workflows/nightly-merge.yml

```diff
@@ -0,0 +1,25 @@
+name: 'Nightly Merge (master to iiot)'
+
+on:
+  schedule:
+    - cron:  '0 0 * * *'
+
+jobs:
+  nightly-merge:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v1
+
+    - name: Nightly Merge
+      uses: robotology/gh-action-nightly-merge@v1.2.0
+      with:
+        stable_branch: 'master'
+        development_branch: 'iiot'
+        allow_ff: false
+        push_token: 'IIOT_NIGHTLY_MERGE_PUSH_TOKEN'
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        IIOT_NIGHTLY_MERGE_PUSH_TOKEN: ${{ secrets.IIOT_NIGHTLY_MERGE_PUSH_TOKEN }}
```

Comment:
Request someone with admin privilege to create a secret "IIOT_NIGHTLY_MERGE_PUSH_TOKEN".  For details, see https://help.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets.

Reply:
We need this push token as all branches on IoT Edge are protected.